packages/api/src/models/db-dev.ts

```diff
@@ -92,6 +93,40 @@ async function createRateLimitTable(ddb: AWS.DynamoDB): Promise<void> {
   }
 }
 
+async function createFeatureFlagsTable(ddb: AWS.DynamoDB): Promise<void> {
+  const doesTableExist = await tableExists(docTableNames.featureFlags, ddb);
+  if (!doesTableExist) {
+    const params: AWS.DynamoDB.CreateTableInput = {
+      AttributeDefinitions: [
+        {
+          AttributeName: partitionKey,
+          AttributeType: "S",
+        },
+        {
+          AttributeName: sortKey,
+          AttributeType: "N",
+        },
```

Comment:
what are S and N? String and Number? Can we give these a name?

Reply:
No, this is how DynamoDB determines types